tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -259,8 +260,16 @@ test.serial('verify that all returned granules have sync_granule_duration set',
 
   await validateConfig(t, t.context.event.config);
   await validateInput(t, t.context.event.input);
-
-  const output = await syncGranule(t.context.event);
+  let output;
+  // this is to try to alleviate a common intermitten failure point in cicd
+  // this will fail if done in a loop to repeatedly re-attempt
+  try {
+    output = await syncGranule(t.context.event);
+  } catch (error) {
+    console.log('known IMF source "verify that all returned granules have sync_granule_duration set" returned a bad value, trying again');
```

Comment:
At a minimum we should be checking that the error *is* the known error and not just *a* error if we're retrying - can we update this to validate the error (also as a form of documentation as to why we're doing this at all)? 

That said, if the lambda is failing units often enough to warrant this test fix, we need to track down why as it might be  happening in production as well.      We shouldn't take a unit test retry approach except with great reluctance *and* a good understanding of why the test environment is different than prod.

Reply:
I went ahead and cut it and scoped it for another PR